### ISSUE TITLE
Merge statement fails when used for processing bigger tables

### ISSUE THREAD

**Issue by wimlauwaert** [dnlnln/generate-sql-merge#75]
Issue:
The merge statement becomes too complex for  MS SQL Server when dealing with bigger tables.

Suggestion:
Enable batch processing (@batchSize parameter) that does a merge per given batch size.

**Comment by hidegh** [comment]
I just had a table with around 40K rows. It works, the trick is to set this:
MSSQL tools / options / query results / results to grix / xml data - Unlimited!

And to restart MSSQL Studio!

**Comment by dnlnln** [comment]
This issue is resolved by #94 and #101. 

Usage:
```
EXEC [WideWorldImporters]..[sp_generate_merge] 
  @schema = 'Application', 
  @table_name='Cities', 
  @max_rows_per_batch = 5000,
  @delete_if_not_matched = 0
```

**Comment by AlexNolasco** [comment]
> I just had a table with around 40K rows. It works, the trick is to set this: MSSQL tools / options / query results / results to grix / xml data - Unlimited!
> 
> And to restart MSSQL Studio!

![image](https://github.com/dnlnln/generate-sql-merge/assets/30424/da01b98b-2577-4d06-91ab-e9b655bcefa4)
